generic_petg.xml.fdm_material

```diff
@@ -0,0 +1,38 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+Generic PLA profile. Serves as an example file, data in this file is not correct.
+-->
+<fdmmaterial xmlns="http://www.ultimaker.com/material">
+    <metadata>
+        <name>
+            <brand>Generic</brand>
+            <material>PETG</material>
+            <color>Generic</color>
+        </name>
+        <GUID></GUID>
```

Comment:
I've generated a GUID for you:

    1cbfaeb3-1906-4b26-b2e7-6f777a8c197a

Since this is generic PETG, rather than a specific product, it doesn't matter what GUID there is as long as it is unique, since there are not going to be any spools with this GUID.